console/static/surveys/023.0102.json

```diff
@@ -13,9 +13,23 @@
     "ru_ref": "12345678901A"
   },
   "data": {
+    "146": "Change comments included",
+    "146a": "Yes",
+    "146b": "In-store / online promotions",
+    "146c": "Special events (e.g. sporting events)",
+    "146d": "Calendar events (e.g. Christmas, Easter, Bank Holiday)",
+    "146e": "Weather",
+    "146f": "Store closures",
+    "146g": "Store openings",
+    "146h": "Other",
     "11": "1/4/2016",
     "12": "31/10/2016",
+    "22": "705000",
```

Comment:
We shouldn't need to add any questions, the refresh should just be the updated comments ie question 146* etc.